.travis.yml

```diff
@@ -25,7 +25,7 @@ addons:
   apt:
     sources:
     - ubuntu-toolchain-r-test
-    - llvm-toolchain-trusty-8
+    - llvm-toolchain-xenial-8
```

Comment:
Once again, please revert merging these branches. They are not intended to be merged.